plugins/optimization-detective/optimization.php

```diff
@@ -139,7 +167,7 @@ function od_is_response_html_content_type(): bool {
  * @return string Filtered template output buffer.
  */
 function od_optimize_template_output_buffer( string $buffer ): string {
-	if ( ! od_is_response_html_content_type() ) {
+	if ( ! od_is_response_html_content_type() ) { // TODO: This should check to see if there is an HTML tag.
```

Comment:
Is this `TODO` intended for this PR or another issue?

Reply:
Yes, I'm opening a sub-PR for this. It's unrelated to output buffering.

Reply:
See https://github.com/WordPress/performance/pull/1442